pkgs/tools/security/wordlists/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, pkgs
+, runtimeShell
+, stdenv
+, symlinkJoin
+, tree
+}:
+
+let
+  _wordlistData =
+    [ { name = "dirbuster"; expressionPath = ./dirbuster.nix; }
+      { name = "nmap"; expressionPath = ./nmap.nix; needsPackage = true; }
+      { name = "rockyou"; expressionPath = ./rockyou.nix; }
+      { name = "seclists"; expressionPath = ./seclists.nix; }
+      { name = "wfuzz"; expressionPath = ./wfuzz.nix; needsPackage = true; }
+    ];
+
+  scopedWordlists = lib.makeScope pkgs.newScope (self: with self;
+    let _packageWithLinkCommandFrom = acc: datum:
+      # Some wordlists are based on an already existing `nixpkgs` package; Pass it as parameter of the expression.
+      let parameters = {}
+        // lib.optionalAttrs ((builtins.elem "needsPackage" (builtins.attrNames datum)) && datum.needsPackage) { ${datum.name} = pkgs.${datum.name}; };
+      in
+        let p = callPackage datum.expressionPath parameters;
+        in acc // { ${datum.name} = p; };
+    in builtins.foldl' _packageWithLinkCommandFrom {} _wordlistData
+  );
+
+in let
+  packageConstructor = { withLists ? null, wordlistPackages ? [] }:
+    stdenv.mkDerivation rec {
+      pname = "wordlists";
+      version = "unstable-2020-11-23";
+
+      dontUnpack = true;
+
+      propagatedBuildInputs = [ tree ];
+
+      symlinks = symlinkJoin {
```

Comment:
NOTE: you could replace the `stdenv.mkDerivation` above with `symlinkJoin`, as symlinkJoin should also accept the classic derivation parameters (this package has nothing to really compile, so it won't need the compile stuff included in `stdenv`).

Reply:
I tried to, but the `installPhase`, during which the `wordlists` and `wordlists_path` commands are defined, does not seem to be running...

Reply:
You would need to replace `installPhase` with `postBuild`, like I've done in https://github.com/Pamplemousse/nixpkgs/pull/1